Nodejs/Product/Npm/SPI/Person.cs

```diff
@@ -71,9 +83,9 @@ internal class Person : IPerson {
             // Verify we are parsing correctly
             try {
                 var jObject = JObject.Parse(source);
-                Debug.Assert(((string)jObject["name"] ?? string.Empty) == Name, string.Format("Failed to parse name from {0}", source));
-                Debug.Assert(((string)jObject["email"] ?? string.Empty) == Email, string.Format("Failed to parse email from {0}", source));
-                Debug.Assert(((string)jObject["url"] ?? string.Empty) == Url, string.Format("Failed to parse url from {0}", source));
+                Debug.Assert((string)jObject["name"] == Name, string.Format("Failed to parse name from {0}", source));

```

Comment:
We don't care unless we're doing it wrong. It's nice to keep this codepath around in an #if DEBUG section and assert we're getting the same results as our own parser in case there are any inputs we're not handling properly (which is how we managed to track down this bug in the first place.)